crates/net/rpc-types/src/eth/transaction/request.rs

```diff
@@ -95,11 +95,11 @@ impl TransactionRequest {
             (None, Some(_), access_list) | (None, None, access_list @ None) => {
                 // Empty fields fall back to the canonical transaction schema.
                 Some(TypedTransactionRequest::EIP1559(EIP1559TransactionRequest {
-                    nonce: nonce.unwrap_or(U256::zero()),
+                    nonce: nonce.unwrap_or(U256::ZERO),
```

Comment:
imo it would make it less painfull if there was a equivalent `const fn zero()` function for ruint U256.

Reply:
supportive but not a blocker imo,given this is a one time change